css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
AFAIK the closest standard definition of "P3D65" is in Annex D of SMPTE ST 2067-21:2016, where it is specified as:
* the color primaries specified in SMPTE RP 431-2, and
* the illuminant D65 specified in SMPTE RP 177.

That definition does not include a transfer function. Moreover, the COLOR.6 colorspace defined in the same specification combines P3D65 colorimetry with the PQ transfer function.

I would therefore not use "P3D65", and instead use "P3D65Gamma_X_X".

Perhaps more importantly, I do not expect content to be distributed using this particular combination of colorimetry and transfer function.